src/main/java/at/hannibal2/skyhanni/test/command/ErrorManager.kt

```diff
@@ -113,18 +113,17 @@ object ErrorManager {
         noStackTrace: Boolean,
         vararg extraData: Pair<String, Any?>,
     ) {
-        val error = Error(message, throwable)
-        error.printStackTrace()
-        Minecraft.getMinecraft().thePlayer ?: return
-
         if (!ignoreErrorCache) {
             val pair = if (throwable.stackTrace.isNotEmpty()) {
-                throwable.stackTrace[0].let { it.fileName!! to it.lineNumber }
+                throwable.stackTrace[0].let { (it.fileName ?: "null") to it.lineNumber }
```

Comment:
I am pretty sure the standard way to present an absent fileName would be `"<unknown>"`